integrationtest/mapper/orm-realbackend/src/test/java/org/hibernate/search/integrationtest/mapper/orm/realbackend/routing/RoutingBridgeRoutingKeyIT.java

```diff
@@ -52,6 +52,10 @@ public void setup() {
 		entityManagerFactory = setupHelper.start()
 				.withProperty( HibernateOrmMapperSettings.AUTOMATIC_INDEXING_SYNCHRONIZATION_STRATEGY,
 						AutomaticIndexingSynchronizationStrategyNames.READ_SYNC )
+				// This call in testLifecycle to Elasticsearch sever may exceed 30 seconds,
+				// from some machine. This is a server (not Hibernate Search) latency.
+				// E.g: > query parameters {} and 1 objects in payload in 30617ms.
+				.withProperty( "hibernate.search.backend.read_timeout", "60000" )
```

Comment:
Come on, this test takes 5 seconds on my machine. No way this takes more than 30s on yours unless there's yet another problem... Let's talk about it on Zulip.